web/src/main/java/com/navercorp/pinpoint/web/service/SpanServiceImpl.java

```diff
@@ -355,6 +355,11 @@ private void transitionException(List<SpanAlign> spanAlignList) {
 
     private StringMetaDataBo selectStringMetaData(String agentId, int cacheId, long agentStartTime) {
         final List<StringMetaDataBo> metaDataList = stringMetaDataDao.getStringMetaData(agentId, agentStartTime, cacheId);
+		if (cacheId == 404) {
```

Comment:
Hi emeroad so we came up with two options we think will make this compatible 

one we could change the random range to be 500 to 5000 rather than 0 to 5000
within com.navercorp.pinpoint.common.server.bo on line 120 exceptionInfo.setIntValue(RandomUtils.nextInt(1000, 5000));

or we could simply change our 404 int on the identifier we are using to 5001 since that is outside the bounds of the current randomly assigned int

We really think having a set range that can be used for custom identifiers is a good approach for this and possible future customized exception options within pinpoint...can you let us know if either of these approaches will work for you all and if so which one?